reaching-the-venues.md

```diff
@@ -12,13 +12,24 @@ This page is to help the attendees reach Chennai and the conference venues.
 ### Getting to PyCon India 2019
 
 <br>
+<div class="img">
+     <img src="./assets/images/reaching-venue19-distance.png" 
```

Comment:
Why is this class needed?

Reply:
When i added markdown image syntax, I am unable to click the image to see in an enlarged view, Then I searched online to see how can I make the image look bigger on click. Then i found one solution is to use transform css property to enlarge the image.